google/cloud/spanner_dbapi/connection.py

```diff
@@ -591,6 +604,7 @@ def connect(
     pool=None,
     user_agent=None,
     client=None,
+    database_dialect=DatabaseDialect.GOOGLE_STANDARD_SQL,
```

Comment:
I think that it would be better if we would automatically read this from the database, instead of requiring the user to supply it as an argument. We already read this property from the database in the `database.default_schema` method.

Reply:
Done